wordpress-mobile/WordPress-Android
# Unified Comments List: basic scaffolding

This is very basic scaffolding for the unified comment list. There is no testable logic at this point, and the PR is made to reduce the review load from the subsequent PR's.

Scaffolding contains:
- Basic activity with comments list fragment.
- Comment list adapter that using `PagingDataAdapter` from the new Paging Library.
- ViewHolders that are mostly copied from exiting implementation (minus some styling code that will go into UI utils).
- ViewModel that links a dummy data source (`CommentPagingSource`) to the adapter.

To test:
- Using wasabi variant navigate to Unified Comments list from My Site screen.
- Confirm that you see a list of 30 unstyled comments and one sub-header at the top.
[![Image from Gyazo](https://i.gyazo.com/31e6373ab1023a3fe8b10df2092483dc.png)](https://gyazo.com/31e6373ab1023a3fe8b10df2092483dc)

## Regression Notes
1. Potential unintended areas of impact
None. The functionality is behind feature flag.

PR submission checklist:
